### DISCUSSION TITLE
Change Cell Size

### DISCUSSION THREAD

**Discussion by Deterez** [Daniel09Fernandes/Component_OpenOffice#33] [Announcements]
**Is your feature request related to a problem? Please describe.**
I want to be able to change width of collumns when i create them, when i fill cells with long text the sheet looks messed up.

**Describe the solution you'd like**
Add functionality to change size of cell just like you add borders etc.

**Comment by Daniel09Fernandes** [comment]
Hello Deterez, I will analyze to include this feature

**Comment by Deterez** [comment]
Thank you

pt., 2 cze 2023, 16:36 użytkownik Daniel Fernandes ***@***.***>
napisał:

**Comment by Daniel09Fernandes** [comment]
Hi @Deterez 
Use the setCell Width to modify the cell width, to modify the height, use the Wrapped property that is in the SetValue function.

function SetValue(aCellNumber: integer; const aCollName: string; aValue: variant; TypeValue: TTypeValue = ftString; Wrapped: boolean = false): TOpenOffice_calc;

**Reply by Deterez** [reply]
Hello.
Just what i needed, thank you very much.

**Reply by Deterez** [reply]
Hi, after trying to use it I've noticed that You implemented it only on normal source, is there a chance for a DX7 version too?

**Reply by Daniel09Fernandes** [reply]
Hello @Deterez, 
Implemented for Delphi 7, same implementation.

Thanks for the feedback 😀

**Reply by Deterez** [reply]
Thank you very much :)

**Reply by Daniel09Fernandes** [reply]
Your welcome :)
